packages/circuit-ui/components/Button/Button.tsx

```diff
@@ -25,6 +25,7 @@ import {
 import { css } from '@emotion/react';
 import isPropValid from '@emotion/is-prop-valid';
 import { Theme } from '@sumup/design-tokens';
+import { isBoolean } from 'lodash/fp';
```

Comment:
We're trying to move away from `lodash` to reduce the number of dependencies. This can be done with a simple `typeof` check.

Reply:
sure thing, I have replaced its usage here.